src/Grok.Net/Grok.cs

```diff
@@ -38,21 +39,34 @@ public Grok(string grokPattern)
         /// </summary>
         /// <param name="grokPattern">The Grok pattern to use.</param>
         /// <param name="customPatterns">A stream containing custom patterns.</param>
-        public Grok(string grokPattern, Stream customPatterns)
-            : this(grokPattern)
+        public Grok(string grokPattern, Stream customPatterns) : this(grokPattern)
         {
-            LoadCustomPatterns(customPatterns);
+            LoadCustomPatternsFromStream(customPatterns);
```

Comment:
I think, if we want to rename the method, we should use a static polymorphism - method overloading.
No need to specify `FromXXX` in the name if it is clear from the param.
What do you think?

Reply:
Makes sense. Will change.